spec/1.x/dispatch.spec.js

```diff
@@ -0,0 +1,81 @@
+var should = require('should');
+var dispatch = require('../../src/js/utils/dispatch');
+
+describe('1.x dispatch', function () {
+
+  var oldDocument = global.document;
+
+  /*
+  after(function () {
+    global.document = oldDocument;
+    global.window = document.defaultView;
+  });
+  */
+
+  it('attaches an event to an element', function () {
+    var element = document.body;
+    var flag = false;
+    var listener = function (e) { flag = true; };
```

Comment:
This could probably be done a little cleaner with a library like sinon.js, where you can use spys. But it's fine for now.